charts/backingservices/README.md

```diff
@@ -34,6 +34,10 @@ Note: Pega does **not** actively update the elasticsearch dependency in `require
 * Update the elasticsearch `version` value, or
 * Disable deploying elasticsearch by setting `srs.srsStorage.provisionInternalESCluster` to false.
 
+## Deploying SRS with Pega-provided busybox images
+The busybox image is required to deploy SRS. For clients who want to pull this image from a registry other than Docker Hub, they must tag and push their image to the registry, and then pull it by specifying `busybox.image` and `busybox.imagePullPolicy`.
```

Comment:
can you update the statement here as well so there's 1-to-1 consistency?? otherwise it looks great! Second sentence is perfect.>>>
The busybox image is required to deploy SRS.
to>
To deploy Pega Platform, the SRS helm chart requires the use of the busybox image.

Reply:
The reason I didn't initially use the same "To deploy Pega Platform..." part is because the SRS chart is independent of the Pega Platform.  So how about

"To deploy the Search and Reporting Service as an independent module, the SRS helm chart requires the use of the busybox image."

Reply:
Or maybe this is better?

"To deploy Pega Platform with the SRS backing service, the SRS helm chart requires the use of the busybox image."

Reply:
sure this is good! thanks!

Reply:
Done! :)